LICENSE.md

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2017-2023 Checkin.com
```

Comment:
Great, but please add the words `The MIT License`  as the heading on top. (similar to https://github.com/stripe/stripe-android/blob/master/LICENSE )

Reply:
Sure, missed that, thanks! @joelcassel